Tasks/PipAuthenticateV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,23 @@
+{
+  "loc.friendlyName": "Python pip authenticate",
+  "loc.helpMarkDown": "[Learn more about this task](https://pip.pypa.io/en/stable/reference/pip_install/) or [see the pip documentation](https://pip.pypa.io/)",
```

Comment:
This doesn't seem right? aka link to our (future) docs?

Reply:
Thanks for catching that. Fixed that